contrib/config/test/kv_store_xds_delegate_integration_test.cc

```diff
@@ -0,0 +1,367 @@
+#include "envoy/admin/v3/config_dump.pb.h"
+#include "envoy/service/runtime/v3/rtds.pb.h"
+#include "envoy/service/secret/v3/sds.pb.h"
+
+#include "test/common/grpc/grpc_client_integration.h"
+#include "test/integration/http_integration.h"
+#include "test/test_common/status_utility.h"
+#include "test/test_common/utility.h"
+
+#include "gtest/gtest.h"
+
+namespace Envoy {
+namespace {
+
+constexpr char SDS_CLUSTER_NAME[] = "sds_cluster.lyft.com";
+constexpr char RTDS_CLUSTER_NAME[] = "rtds_cluster";
+constexpr char CLIENT_CERT_NAME[] = "client_cert";
+
+std::string kvStoreDelegateConfig() {
+  const std::string filename = TestEnvironment::temporaryPath("xds_kv_store.txt");
+  ::unlink(filename.c_str());
```

Comment:
Can this use [this unlink](https://github.com/envoyproxy/envoy/blob/main/envoy/api/os_sys_calls.h#L212)?

Reply:
done, and for the one below